Source/Particles/Collision/MCCScattering.H

```diff
@@ -0,0 +1,304 @@
+/* Copyright 2021 Modern Electron
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_PARTICLES_COLLISION_MCC_SCATTERING_H_
+#define WARPX_PARTICLES_COLLISION_MCC_SCATTERING_H_
+
+#include "MCCProcess.H"
+#include "Utils/WarpXConst.H"
+#include <AMReX_Random.H>
+
+/** @file
+ *
+ * This file contains the implementation of the scattering processes available
+ * in the MCC handling.
+ */
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void getRandomVector ( amrex::Real& x, amrex::Real& y, amrex::Real& z,
+                       amrex::RandomEngine const& engine )
+{
+    using std::sqrt;
+    using std::cos;
+    using std::sin;
+
+    // generate random unit vector in 3 dimensions
+    // https://math.stackexchange.com/questions/44689/how-to-find-a-random-axis-or-unit-vector-in-3d
```

Comment:
```suggestion
    // https://mathworld.wolfram.com/SpherePointPicking.html
```
Minor change to reflect the original link that was discussed in this post.